src/components/calculator/PrevalenceControls.tsx

```diff
@@ -249,78 +257,98 @@ export const PrevalenceControls: React.FunctionComponent<{
           </select>
         </div>
       )}
-
+      {!showEnterDataButton ? (
+        <button
+          id="switchBetweenManualDataAndLocationSelection"
+          className="btn btn-secondary"
+          onClick={handleSelectLocationButtonOnClick}
+        >
+          {t('calculator.switch_button.select_location')}
+        </button>
+      ) : (
+        <button
+          id="switchBetweenManualDataAndLocationSelection"
+          className="btn btn-secondary"
+          onClick={handleEnterDataButtonOnClick}
+        >
+          {t('calculator.switch_button.enter_data_manually')}
+        </button>
+      )}
       <ControlledExpandable
-        id="prevelance-details"
+        id="prevalence-details"
         header={t('calculator.prevalence.details_header')}
         headerClassName={isManualEntryCurrently ? 'd-none' : ''}
         open={detailsOpen}
         setter={setDetailsOpen}
       >
-        {!isManualEntryCurrently && <PrevalenceResult data={data} />}
-
-        <PrevalenceField
-          id="reported-cases"
-          label={t('calculator.prevalence.last_week_cases')}
-          value={data.casesPastWeek || 0}
-          setter={(value) =>
-            setter({ ...data, casesPastWeek: parseInt(value || '') })
-          }
-          inputType="number"
-          isEditable={!locationSet}
-          className="hide-number-buttons"
-        />
-        <PrevalenceField
-          id="population"
-          label={t('calculator.prevalence.population')}
-          value={data.population}
-          setter={(value) => setter({ ...data, population: value })}
-          inputType="text"
-          isEditable={!locationSet}
-          className="hide-number-buttons"
-        />
-        {locationSet && data.casesIncreasingPercentage === 0 ? (
-          <div>{t('calculator.prevalence.cases_stable_or_decreasing')}</div>
-        ) : (
-          <PrevalenceField
-            id="precent-increase"
-            label={t('calculator.prevalence.percent_increase_in_cases')}
-            value={data.casesIncreasingPercentage}
-            unit="%"
-            setter={(value) => {
-              setter({ ...data, casesIncreasingPercentage: Number(value) })
-            }}
-            inputType="number"
-            min={0}
-            isEditable={!locationSet}
-            className="hide-number-buttons"
-          />
-        )}
-        {data.positiveCasePercentage === null ? (
-          <PrevalenceField
-            id="positive-test-rate"
-            label={t('calculator.prevalence.positive_case_percentage')}
-            value="no data available"
-            unit="%"
-            setter={(_value) => null}
-            inputType="text"
-            isEditable={false}
-          />
+        {!isManualEntryCurrently ? (
+          <PrevalenceResult data={data} />
         ) : (
-          <PrevalenceField
-            id="positive-test-rate"
```

Comment:
I'd still like to see these fields show up even if automatic data location is being used:

![image](https://user-images.githubusercontent.com/32290/104152393-41415d00-5394-11eb-8318-89ed800922ca.png)